specification/paths/SuggestAddress.json

```diff
@@ -25,9 +25,6 @@
             "properties": {
               "data": {
                 "type": "object",
-                "required": [
-                  "country_code"
-                ],
```

Comment:
FYI I am not entirely sure if it is possible to do comprehensive required rules like with Laravel, therefore I removed the required constraint here.